src/Model_Hydro/CPU_Hydro/CPU_Shared_RiemannSolver_Exact.cpp

```diff
@@ -88,9 +89,9 @@ void Hydro_RiemannSolver_Exact( const int XYZ, real Flux_Out[], const real L_In[
 
 // convert conserved variables to primitive variables
    Hydro_Con2Pri( L_In, L, MinPres, FracPassive_No, NULL_INT, NULL, JeansMinPres_No, NULL_REAL,
-                  EoS_DensEint2Pres, NULL, EoS_AuxArray_Flt, EoS_AuxArray_Int, EoS_Table, NULL );
+                  EoS_DensEint2Pres, NULL, NULL, NULL, EoS_AuxArray_Flt, EoS_AuxArray_Int, EoS_Table, NULL, NULL );
```

Comment:
The `EoS_GuessHTilde`, `EoS_HTilde2Temp`, and `LorentzFactorPtr` should be `NULL` if `SRHD` is not enabled. Should we replace the `NULL` to be the variables respectively? (Same question to the other `NULL`s)